core/src/main/java/nl/nn/adapterframework/webcontrol/api/BrowseQueue.java

```diff
@@ -54,10 +54,11 @@
 	@Produces(MediaType.APPLICATION_JSON)
 	public Response getBrowseQueue() throws ApiException {
 		Map<String, Object> returnMap = new HashMap<String, Object>();
+		JmsBrowser<javax.jms.Message> jmsBrowser = getIbisContext().createBeanAutowireByName(JmsBrowser.class);
 
-		List<String> jmsRealms=JmsRealmFactory.getInstance().getRegisteredRealmNamesAsList();
-		if (jmsRealms.size()==0) jmsRealms.add("no realms defined");
-		returnMap.put("jmsRealms", jmsRealms);
+		List<String> connectionFactories=jmsBrowser.getConnectionFactoryFactory().getConnectionFactoryNames();
+		if (connectionFactories.size()==0) connectionFactories.add("no connection factories found");
+		returnMap.put("connectionFactories", connectionFactories);
```

Comment:
It think it would be the best to extending this list of connectionFactories with that found in the jmsRealms (avoiding duplicates). That allows to use connectionFactories defined in a jmsRealm, but not yet in a real configuration element.